### PR TITLE
Fix dependency vulnerabilities alert

### PR DESCRIPTION
**Pull request by yec-akamai**
## 📝 Description

Fixed a couple of critical vulnerabilities by 
- Bumping babel/core and babel/traverse to 7.23.2
- Bump trim-newlines to 4.0.1
- Bump send to 0.19.0

Should address https://github.com/linode/rancher-ui-driver-linode/security/dependabot/42, https://github.com/linode/rancher-ui-driver-linode/security/dependabot/46, and https://github.com/linode/rancher-ui-driver-linode/security/dependabot/14.

## ✔️ How to Test

1. Run `npm install` against main branch and see a couple of vulnerabilities found.
2. Pull this branch and run `npm install`.
3. No vulnerabilities found.
4. Run `npm run build` and no error raised.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,10 +29,10 @@
     "gulp-bump": "^3.2.0",
     "gulp-clean": "0.4.0",
     "gulp-concat": "2.6.1",
-    "gulp-connect": "5.7.0",
+    "gulp-connect": "^5.0.0",
     "gulp-jshint": "2.1.0",
     "gulp-replace": "1.0.0",
-    "gulp-sourcemaps": "3.0.0",
```
Comment (zliang-akamai) [inline package.json]: This is a downgrade?

**Comment by zliang-akamai** [comment]
Have you tried testing the UI against a rancher instance?

File: package.json (hunk 1 above)
Comment (yec-akamai) [reply package.json]: It was changed by `npm audit fix` to fix some vulnerabilities automatically.

**Comment by yec-akamai** [comment]
> Have you tried testing the UI against a rancher instance?

Yes, deployed and built the ui driver worked well.

**Comment by yec-akamai** [comment]
Revert the downgrading of gulp-connect,gulp-sourcemaps. It leaves a moderate severity vulnerability `PostCSS line return parsing error`, which is addressed in 
 https://github.com/linode/rancher-ui-driver-linode/pull/67/files

**Review by zliang-akamai — APPROVED** [review]
LGTM!

**Review by lgarber-akamai — APPROVED** [review]
Everything works as expected on my end, nice work!

**Pull request merged**